### DISCUSSION TITLE
Alias categorization

### DISCUSSION THREAD

**Discussion by patriksvensson** [cake-build/cake#3539] [Ideas]
The number of categories in the [reference section](http://cakebuild.net/dsl) are growing and should be better organized.

**Proposal:**
- Build Tools
  - MSBuild
  - XBuild
  - DotNetBuild
- CI Servers
  - AppVeyor
  - TeamCity
- Environment
  - Arguments
  - Platform
  - Variables
- Net
  - HTTP Operations
- Logging
- IO
  - Compression
  - Directories
  - Files
  - Globbing
  - Processes
- Security
  - Hashing
  - Signing
- Test Frameworks
  - Fixie
  - MSTest
  - NUnit
  - xUnit
  - xUnit v2
- Text
  - Release Notes
  - Transformation
- Tools
  - ILMerge
  - NSIS
  - NuGet
  - Octopus Deploy
  - Roundhouse
  - WiX
- XML
  - Transform

**Comment by ChrisMissal** [comment]
I just have one suggestion: Fixie should be moved under Test Frameworks

**Comment by patriksvensson** [comment]
You are right! Not sure how I missed that. Thanks!

**Comment by patriksvensson** [comment]
Updated :smile:
